Dokploy LXC update script missing "update" argument

### ✅ Have you read and understood the above guidelines?

Yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

ProxMoxVE/ct/dokploy.sh

### 📂 What was the exact command used to execute the script?

update

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 📈 Which Proxmox version are you on?

pve-manager/8.4.14/b502d23c55afcba1 (running kernel: 6.8.12-15-pve)

### 📝 Provide a clear and concise description of the issue.

[Line 33](https://github.com/community-scripts/ProxmoxVE/blob/53f89258a4fa45af154d9ad927dc56714495946b/ct/dokploy.sh#L33) of the Dokploy update script is incomplete. According to [Dokploy's Manual Upgrade documentation](https://docs.dokploy.com/docs/core/manual-installation#manual-upgrade), the correct command requires the user to pass `update` as an argument. Without this argument, the `curl`ed Dokploy script attempts a fresh install of Dokploy, and errors out, as the required port is unavailable.

The current command: `curl -sSL https://dokploy.com/install.sh`
The correct command:  `curl -sSL https://dokploy.com/install.sh | sh -s update`

Running the correct command in the Dokploy LXC console successfully updates Dokploy.

### 🔄 Steps to reproduce the issue.

Step 1: Install Dokploy LXC, default options, accept USB passthrough
Step 2: Run `update` command from Dokploy LXC console

### ❌ Paste the full error output (if available).

```
root@dokploy:~# update
  ✖️   in line 33: exit code 1 (General error / Operation not permitted)
 → bash /dev/fd/63

  --- Last 10 lines of log ---
[2026-02-20 11:43:53] [INFO] Updating Dokploy
Detecting latest stable version from GitHub...
Latest stable version detected: v0.27.1
Installing Dokploy version: v0.27.1
Error: something is already running on port 80
[2026-02-20 11:43:54] [ERROR] in line 33: exit code 1 (General error / Operation not permitted)
[2026-02-20 11:43:54] bash /dev/fd/63
  -----------------------------------
```

### 🖼️ Additional context (optional).

_No response_

--- a/ct/dokploy.sh
+++ b/ct/dokploy.sh
@@ -30,7 +30,7 @@ function update_script() {
   fi
 
   msg_info "Updating Dokploy"
-  $STD bash <(curl -sSL https://dokploy.com/install.sh)
+  curl -sSL https://dokploy.com/install.sh | $STD bash -s update
   msg_ok "Updated Dokploy"
   msg_ok "Updated successfully!"
   exit

